### ISSUE TITLE
Allow libzim access on Android by default

### ISSUE THREAD

**Issue by Jaifroid** [kiwix/kiwix-js-pwa#461]
When #459 is complete, it will be feasible to allow libzim access by default in the PWA on Android. Until now, this access has been painfully slow to the point of being unusable. Although some users had requested an expert setting to enable it for use on Android tablets, and I had provided this, it remained off by default, and hence full-text search is disabled.

However, the OPFS provides something like 10X the speed of access, and for the size of archive users are likely to store there, access to full-text search is finally useable. Therefore, the default should be changed to allow use.

**Comment by Jaifroid** [comment]
#459 Includes support for libzim access for archives in the OPFS. I am still considering whether it should be the global default.
